images/macos/provision/core/gnu-tar.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e -o pipefail
+
+echo "Installing gnu-tar..."
+brew install gnu-tar
+
+echo 'Updating PATH to use gnu-tar as "tar"'
+export PATH="/usr/local/opt/gnu-tar/libexec/gnubin:$PATH"
```

Comment:
I think we mustn't replace default tar by GNU-Tar. It could cause unpredictable impact to customers.
cc: @al-cheb @miketimofeev any thoughts?

Reply:
@maxim-lobanov Absolutely agree

Reply:
As I know, gnu tar should be available as `gtar` after installation. It should be enough

Reply:
Fixed it